frontend/common/Status.tsx

```diff
@@ -89,6 +89,7 @@ function useLabel(status: string | undefined, t: (str: string) => string) {
     unknown: t('Unknown'),
     unreachable: t('Unreachable'),
     waiting: t('Waiting'),
+    'workers offline': t('Workers offline'),
```

Comment:
```suggestion
    'workers-offline': t('Workers offline'),
```

Just to be consistent with the remaining work.